com.unity.formats.fbx/Editor/FbxPropertyChannelPair.cs

```diff
@@ -31,122 +31,150 @@ public UnityPropertyChannelPair(string p, string c)
         }
 
         /// <summary>
-        /// Contains the two dictionaries that map Unity property to FBX property and Unity channel to Fbx channel
+        /// Contains the two lists that map Unity property to FBX property and Unity channel to Fbx channel
         /// for a set of properties.
         /// </summary>
         struct PropertyChannelMap
         {
-            public Dictionary<string, string> MapUnityPropToFbxProp;
-            public Dictionary<string, string> MapUnityChannelToFbxChannel;
+            public List<(string, string)> MapUnityPropToFbxProp;
+            public List<(string, string)> MapUnityChannelToFbxChannel;
 
-            public PropertyChannelMap(Dictionary<string,string> propertyMap, Dictionary<string, string> channelMap)
+            public PropertyChannelMap(List<(string, string)> propertyMap, List<(string, string)> channelMap)
             {
                 MapUnityPropToFbxProp = propertyMap;
                 MapUnityChannelToFbxChannel = channelMap;
             }
+
+            private string GetFbxValue(string uniValue, List<(string,string)> list)
+            {
+                var index = list.FindIndex(x => x.Item1 == uniValue);
+                return index < 0 ? null : list[index].Item2;
```

Comment:
does `return list.FirstOrDefault(x => x.Item1 == uniValue).Item2` work? 

If not then this is good too.

Reply:
there isn't a first or default, but I think Find gives the same result